images/win/scripts/Installers/Install-Firefox.ps1

```diff
@@ -5,71 +5,48 @@
 
 Import-Module -Name ImageHelpers -Force
 
-$temp_install_dir = 'C:\Windows\Installer'
-New-Item -Path $temp_install_dir -ItemType Directory -Force
-
-$versionsJson = Invoke-RestMethod  "https://product-details.mozilla.org/1.0/firefox_versions.json"
-$latestVersion = $versionsJson.LATEST_FIREFOX_VERSION
-Write-Host "Firefox latest version: $latestVersion"
-
-# url for latest version of firefox
-$urlLatestVersion = "https://download.mozilla.org/?product=firefox-${latestVersion}&os=win64&lang=en-US"
-Install-Binary -Url $urlLatestVersion -Name "Firefox Setup $latestVersion.exe" -ArgumentList ("/silent", "/install")
-
-# Disable autoupdate
-$firefoxDirectoryPath = Join-Path $env:ProgramFiles "Mozilla Firefox"
-New-Item -path $firefoxDirectoryPath -Name 'mozilla.cfg' -Value '//
+# Install and configure Firefox browser
+Write-Host "Install latest Firefox browser..."
+$VersionsManifest = Invoke-RestMethod "https://product-details.mozilla.org/1.0/firefox_versions.json"
+$InstallerName = "firefox-browser.exe"
+$InstallerUrl = "https://download.mozilla.org/?product=firefox-$($VersionsManifest.LATEST_FIREFOX_VERSION)&os=win64&lang=en-US"
```

Comment:
Looks like we can create new variable `$LatestFirefoxVersion = $VersionsManifest.LATEST_FIREFOX_VERSION` and use it here. It'd be more readable